api/diffs.diff

```diff
@@ -1,14 +1,14 @@
---- 5.3.0f4.h	2020-07-19 20:23:20.846734000 +0200
-+++ 5.3.1f1.h	2020-07-19 20:23:20.889741400 +0200
```

Comment:
Originally the diffs were just used for testing what had changed, but now they’re useful for documenting changes across versions. But the time stamps are too noisy. I wonder if there’s a nice way to strip them - maybe `filterdiff --remove-timestamps` if that’s installed, or some `sed` magic. (Doesn’t have to go in to this commit - I’m just thinking out loud).

Reply:
The timestamps were irking me a bit too, it was just a low priority for the moment - but I agree.